src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/SubstractionConverter.cs

```diff
@@ -23,11 +23,19 @@ public class SubstractionConverter : IMultiValueConverter
     {
         public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture)
         {
-            if (values[0] == DependencyProperty.UnsetValue || values[1] == DependencyProperty.UnsetValue || values[2] == DependencyProperty.UnsetValue)
+            if (values != null
+                && values.Length == 3
+                && (values[0] == DependencyProperty.UnsetValue || values[1] == DependencyProperty.UnsetValue || values[2] == DependencyProperty.UnsetValue))
```

Comment:
Suggestion: to provide a more solid validation, you can try validate more cases in the same if statement.

```suggestion
            if (values == null
                || values.Length < 3
                || values[0] == DependencyProperty.UnsetValue || values[1] == DependencyProperty.UnsetValue || values[2] == DependencyProperty.UnsetValue)
```